src/pvrclient-argustv.cpp

```diff
@@ -1492,7 +1505,33 @@ bool cPVRClientArgusTV::FindRecEntryUNC(const std::string &recId, std::string &r
   if (iter == m_RecordingsMap.end())
     return false;
 
-  recEntryURL = ToUNC(iter->second.c_str());
+  recEntryURL = ToUNC(iter->second);
+  if (recEntryURL == "")
+    return false;
+
+  return true;
```

Comment:
You can easily combine the above four lines into one: `return !recEntryURL.empty();`